docs/content/en/docs/tasks/cluster/cluster-flux.md

```diff
@@ -197,4 +196,80 @@ After your cluster has been created, you can test the GitOps controller by modif
     ```bash
     kubectl get nodes 
     ```
-   
+
+## Getting Started with EKS Anywhere GitOps with any Git source
+You can configure EKS Anywhere to use a generic git repository as the source of truth for GitOps by providing a `FluxConfig` with a `git` configuration.
+
+EKS Anywhere requires a valid SSH Known Hosts file and SSH Private key in order to connect to your repository and bootstrap Flux.
+
+## Create a Git repository for use by EKS Anywhere and Flux
+When using the `git` provider, EKS Anywhere requires that the configuration repository be pre-initialized.
+You may re-use an existing repo or use the same repo for multiple management clusters.
+
+Create the repository through your git provider and initialize it with a `README.md` documenting the purpose of the repository.
+
+## Create a Private Key for use by EKS Anywhere and Flux
+EKS Anywhere requires a private key to authenticate to your git repository, push the cluster configuration, and configure Flux for ongoing management and monitoring of that configuration.
+The private key should have permissions to read and write from the repository in question.
+
+It is recommended that you create a new private key for use exclusively by EKS Anywhere. 
+You can use `ssh-keygen` to generate a new key.
+
+```sh
+ssh-keygen -t ecdsa -C "my_email@example.com"
+```
+
+Please consult the documentation for your git provider to determine how to add your key; for example, you can find the documentation 
```

Comment:
Were you planning to add a link to documentation for adding keys here?

Reply:
yes, let me add that